packages/fake-data/src/fake/fake.ts

```diff
@@ -214,7 +215,8 @@ export function fakeEncounterMedication(prefix = 'test-') {
 }
 
 export const fakeDate = () => chance.date();
-export const fakeString = (model, { fieldName }, id) => `${model.name}.${fieldName}.${id}`;
+export const fakeString = (model: typeof Model, { fieldName }, id: string) =>
```

Comment:
```suggestion
export const fakeString = (model: typeof Model, { fieldName } : { fieldName: string } , id: string) =>
```
I dunno actuly probably shud be typeof attributes sumhow but minor and just skip for now